src/main/java/bio/terra/common/exception/stairway/StairwayInitializationException.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.common.exception.stairway;
+
+import bio.terra.stairway.exception.StairwayRuntimeException;
+
+public class StairwayInitializationException extends StairwayRuntimeException {
```

Comment:
Seems kind of funny to be subclassing a stairway exception.

Reply:
reverted